Spec/Auth.swift

```diff
@@ -2441,6 +2441,26 @@ class Auth : QuickSpec {
                     }
                 }
 
+                it("example: if is initialised with custom ttl and a TokenParams object is passed with null, then the @ttl@ used for every subsequent authorization will be null") {
+                    let options = AblyTests.commonAppSetup()
+                    options.defaultTokenParams = { $0.ttl = 0.1; return $0 }(ARTTokenParams())
+                    let rest = ARTRest(options: options)
+
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.authorise(nil, options: nil) { tokenDetails, error in
```

Comment:
The spec says: "and a TokenParams object is passed in as an argument to #authorize with a null or missing value for ttl" but this is using a null TokenParams. It's the TTL which should be null or missing. Of course, that's not possible in iOS, so you should probably use other TokenParams field which can be null, or define zero as "missing" for the TTL field and test that.

This is also missing the "every subsequent authorization" part. You're testing no subsequent authorizations; at least one would be good.